### PR TITLE
add link to Equifax and TransUnion on Information page 

### LINKED ISSUE #1429: Add links to Information Page tips
## Summary

Add links to Equifax and TransUnion on Information page

                      <A
                        href={
                          i18n.locale === 'en'
                            ? 'https://www.consumer.equifax.ca/personal/'
                            : 'https://www.consumer.equifax.ca/personnel/'
                        }
                        isExternal // Opens new tab
                      />
                      <A
                        href={
                          i18n.locale === 'en'
                            ? 'https://www.transunion.ca/'
                            : 'https://www.transunion.ca/fr'
                        }
                        isExternal // Opens new tab
                      />

** I've tried adding <0></0> and <1></1> in the lingui files but this takes out all spacing around the links.

## Design Detail

Screen Shot 2020-02-18 at 5 22 48 PM

### PR DESCRIPTION
**Pull request by DianeLiu2019**
Fixes #1429 (issue)

# Description
Add links to Equifax and TransUnion on Information page 

# Checklist:
- [ x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)

### CODE REVIEW

**Comment by anikbrazeau** [comment]
Is it possible to center the text within the box?
I'm seeing this on Chrome: 

Screen Shot 2020-02-27 at 9 17 26 AM

**Comment by anikbrazeau** [comment]
Is it possible to center the text within the box?
I'm seeing this on Chrome: 

Screen Shot 2020-02-27 at 9 17 26 AM

File: f2/src/forms/InformationForm.js [hunk 1]
```diff
@@ -146,7 +148,26 @@ export const InformationForm = props => {
             </Field>
             <Alert status="success" backgroundColor="blue.100">
               <AlertIcon name="info-outline" color="blue.800" />
-              <Trans id="informationPage.tip" />
+              <P fontSize="md">
```
Comment (amazingphilippe) [inline f2/src/forms/InformationForm.js]: Remove P to get centered text

Comment (DianeLiu2019) [reply f2/src/forms/InformationForm.js]: fixed add mb={0}

**Pull request merged**